lib/reverse_markdown/converters/li.rb

```diff
@@ -2,10 +2,14 @@ module ReverseMarkdown
   module Converters
     class Li < Base
       def convert(node, state = {})
-        content     = treat_children(node, state)
-        indentation = indentation_from(state)
-        prefix      = prefix_for(node)
-        "#{indentation}#{prefix}#{content.chomp}\n"
+        contains_child_paragraph = node.children.first&.name == 'p'
```

Comment:
Let's try do keep it downward compatible with only a little more work.

```suggestion
        contains_child_paragraph = node.children.first ? node.children.first.name == 'p' : false
```